content/docs/index.mdx

```diff
@@ -29,8 +29,8 @@ hero:
 banner:
   content: |
     <p>
-      This is a live draft of the <strong>Data Package (v2) standard</strong>.
-      Please read the <a href="/blog/2023-11-15-v2-announcement">announcement</a>.
+      The <strong>Data Package (v2)</strong> standard has been released on June 26, 2024!
+      Please read the <a href="/blog/2024-06-26-v2-release">announcement</a>
```

Comment:
```suggestion
      See the <a href="/blog/2024-06-26-v2-release">announcement</a> and <a href="/overview/changelog">changelog</a> for details.
```

If you remove the changelog link, please still include the period at the end. :-)